### PR TITLE
vulkaninfo: Add extension output

### PR DESCRIPTION
**Pull request by jeremyk-lunarg**
Added output for extension structures:

VkPhysicalDevice16BitStorageFeaturesKHR
VkPhysicalDeviceSamplerYcbcrConversionFeaturesKHR
VkPhysicalDeviceVariablePointerFeaturesKHR
VkPhysicalDeviceBlendOperationAdvancedFeaturesEXT
VkPhysicalDeviceBlendOperationAdvancedProper-
tiesEXT
VkPhysicalDevicePointClippingPropertiesKHR
VkPhysicalDevicePushDescriptorPropertiesKHR
VkPhysicalDeviceDiscardRectanglePropertiesEXT

### CODE REVIEW

File: demos/vulkaninfo.c [hunk 1]
```diff
@@ -858,17 +883,38 @@ static void AppGpuInit(struct AppGpu *gpu, struct AppInstance *inst, uint32_t id
...
         if (!queue_priorities) {
             ERR_EXIT(VK_ERROR_OUT_OF_HOST_MEMORY);
         }
-        memset(queue_priorities, 0, gpu->queue_props[i].queueCount * sizeof(float));
+        
+        if (CheckExtensionEnabled(VK_KHR_GET_PHYSICAL_DEVICE_PROPERTIES_2_EXTENSION_NAME, gpu->inst->inst_extensions,
+            gpu->inst->inst_extensions_count)) {
+            memset(queue_priorities, 0, gpu->queue_props2[i].queueFamilyProperties.queueCount * sizeof(float));
+        }
+        else {
```
Comment (mark-lunarg) [inline demos/vulkaninfo.c]: Your clang-format tools is busted.

**Comment by mikew-lunarg** [comment]
@jeremyk-lunarg @shannon-lunarg  Suggestion: there should be tests for vulkaninfo output.  The JSON output has strict formatting requirements, and must validate against a JSON schema.  There should also be a test of HTML validity.

**Comment by jeremyk-lunarg** [comment]
@mikew-lunarg, that sounds like a good idea, but I think we should leave that for a future PR.

File: demos/vulkaninfo.c (hunk 1 above)
Comment (jzulauf-lunarg) [reply demos/vulkaninfo.c]: Check the travis output, several other formatting issues.

**Comment by jeremyk-lunarg** [comment]
Updated

**Comment by jeremyk-lunarg** [comment]
Updated.  Json output removed for the structures mentioned by @shannon-lunarg.

**Pull request merged**